root/etc/services.d/coredns/run

```diff
@@ -2,5 +2,10 @@
 
 cd /config/coredns
 
-exec \
-    /app/coredns -dns.port=53
\ No newline at end of file
+if netstat -apn | grep -q ":53 "; then
```

Comment:
lgtm: :+1: - this addresses the use case of server mode & docker host networking if pihole is already hogging the port :)